ios/FluentUI/People Picker/PeoplePicker.swift

```diff
@@ -88,6 +88,11 @@ open class PeoplePicker: BadgeField {
         }
     }
 
+    /// The UIView used to present 'suggestedPersonas'
```

Comment:
>present [](start = 27, length = 7)

can you add a few more details about how this differs from personaListView

Reply:
added